Sources/Features/Metrics/Model/SummedEvent.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+
+struct SummedEvent: Event {
+
+    var timestamp = Int64()
+    var attributes = DecodableAttributes()
```

Comment:
Those properties are always set in the initializer

```suggestion
    let timestamp: Int64
    let attributes: DecodableAttributes
```

Reply:
Will do, thank you!

Reply:
I will defer initialization to the initializer but in this case as well in the future timestamp and attributes needs to be mutable for retry logic